src/Sylius/Bundle/CoreBundle/Resources/config/services.xml

```diff
@@ -410,6 +410,7 @@
         <service id="sylius.promotion_action.add_product" class="%sylius.promotion_action.add_product.class%">
             <argument type="service" id="sylius.factory.order_item" />
             <argument type="service" id="sylius.repository.product_variant" />
+            <argument type="service" id="sylius.modifier.order_item_quantity" />
```

Comment:
I think we should name it just ``sylius.order_item_quantity_modifier``. Why? Because we should only group services, which really do the same thing. It is true for factory, repository, manager, controller etc. But Modifier in Sylius terms is very vague. We don't have any other modifiers and I doubt we will have anytime soon and if yes, then there is a low chance that they will be doing similar thing to this one. :) WDYT? /cc @michalmarcinkowski